### ISSUE TITLE
Misleading comment

### ISSUE THREAD

**Issue by kstrafe** [robert-w-gries/rust_workshop#1]
In examples/ownership.rs, shouldn't `// Ownership rules do not apply to values stored in stack` be `// Things that are "Copy" are implicitly copied,..` or something along those lines? You can certainly craft a unit struct that lies solely in the stack which does not implement `Copy`, yet to which ownership rules apply.

**Comment by robert-w-gries** [comment]
I agree we can improve the wording of the comment. Maybe we can mention the concept of `copy semantics` replacing `move semantics` when the `Copy` trait is implemented. If you want, please feel free to submit a PR to improve the wording.
